flask_project/campaign_manager/test/test_models.py

```diff
@@ -363,80 +281,69 @@ def test_01_create_task_boundary(self):
                     ]
                 }
         geom_obj = from_shape(asShape(geom), srid=4326)
-        mock_user = get_mock_user(
-            "user",
-            "email@gmail.com"
+        mock_user = User(
+            osm_user_id="user_new",
+            email="email@gmail.com"
             )
-        mock_campaign = get_mock_campaign(
-            mock_user.id,
-            "campaign",
-            "campaign_description",
-            datetime.date(2018, 1, 1),
-            datetime.date(2018, 1, 1),
-            datetime.datetime.now()
+        mock_user.create()
+        mock_campaign = Campaign(
+            creator_id=mock_user.id,
+            name="campaign_new",
+            description="campaign_description",
+            start_date=datetime.date(2018, 1, 1),
+            end_date=datetime.date(2018, 1, 1),
+            create_on=datetime.datetime.now(),
+            uuid="mock_campaign_uuid"
             )
-        mock_task_boundary = get_mock_task_boundary(
-            geom_obj,
-            "Area1",
-            "inactive",
-            "Feature",
-            mock_campaign.id
+        mock_campaign.create()
+        mock_task_boundary = TaskBoundary(
+            coordinates=geom_obj,
+            name="Area1",
+            status="inactive",
+            type_boundary="Feature",
+            campaign_id=mock_campaign.id
             )
-        mock_task_boundary = session.query(
+        mock_task_boundary.create()
+        coor_geoJSON = session.query(
             TaskBoundary.coordinates.ST_AsGeoJSON()
             ).first()
-        geom_check = (geom == ast.literal_eval(mock_task_boundary[0]))
+        geom_check = (geom == ast.literal_eval(coor_geoJSON[0]))
         self.assertTrue(geom_check)
 
     def test_02_task_boundary_geom_type(self):
         import ast
-        mock_campaign = session.query(Campaign).first()
-        mock_geom_obj = session.query(
-            TaskBoundary.coordinates.ST_AsGeoJSON()).filter(
-            TaskBoundary.campaign_id == mock_campaign.id
-            ).first()
+        mock_campaign = campaign.get_by_uuid("mock_campaign_uuid")
+        mock_geom_obj = mock_campaign.get_task_boundary_as_geoJSON()
         mock_geom_obj = ast.literal_eval(mock_geom_obj[0])
         self.assertEqual(mock_geom_obj['type'], "Polygon")
 
     def test_03_task_boundary_campaign_relation(self):
-        mock_task_boundary = session.query(TaskBoundary).first()
-        mock_user = session.query(User).first()
-        mock_campaign = session.query(Campaign).first()
+        mock_task_boundary = task_boundary.get_first()
+        mock_user = user.get_by_osm_id("user_new")
+        mock_campaign = campaign.get_by_uuid("mock_campaign_uuid")
         mock_task_boundary.campaign_id = mock_campaign.id
         session.commit()
```

Comment:
this session.commit() should not be here.

Reply:
I have corrected it.